templates/slurm_logrotate.j2

```diff
@@ -9,7 +9,7 @@
 	compress
 	create 644 slurm slurm
 	postrotate
-		/bin/kill {{ slurm_logrotate_signal }} $(cat /var/run/slurmdbd.pid) 2>/dev/null 2> /dev/null || true
+		/bin/kill {{ slurm_logrotate_signal }} $([[ -r /var/run/slurmdbd.pid ]] && cat /var/run/slurmdbd.pid) 2>/dev/null 2> /dev/null || true
```

Comment:
Cool!

Haven't used that 

```
[[ -r /path ]] 
```

before. Seems to do what we want.
Another reason why the file might not be there I suppose would be if one of these have been changed from the defaults:

```
$ grep -ri pid defaults/main.yml 
slurm_SlurmctldPidFile: "/var/run/slurmctld.pid"
slurm_SlurmdPidFile: "/var/run/slurmd.pid"
```

Would you like to update the PR to also use these variables or shall we go ahead and merge as is?